### ISSUE TITLE
Keyboard navigation and Live Updating Demo sections are missing in Data Grid topic for Korean version.

### ISSUE THREAD

**Issue by HristoP96** [IgniteUI/igniteui-docfx#848]
Before the [**Known Limitation**](https://staging.infragistics.local/products/ignite-ui-angular/angular/components/grid.html#known-limitations) section in the Data Grid topic there are two sections that are omitted in the KR version, which are[ **Live Updating Demo** ](https://staging.infragistics.local/products/ignite-ui-angular/angular/components/grid.html#live-updating-demo) and [**Keyboard Navigation**](https://staging.infragistics.local/products/ignite-ui-angular/angular/components/grid.html#keyboard-navigation)

**Comment by yoshimis** [comment]
@HristoP96 we won't be updating grid and chart related topics in KR if translation involved. We will update them on next release which scheduled for 19.1 next year.

**Comment by HristoP96** [comment]
Ok @yoshimis . I will close this issue then.
